### ISSUE TITLE
font-awesome css online loaded when page or post edited by Elementor

### ISSUE THREAD

**Issue by ryanlabelle** [ryanlabelle/th-widget-pack#155]
Hi @adiraomj when you have time, can you check this out? When viewing a page or psot (not edited with Elementor) the font awesome is not loaded. Kinda makes sense that Elementor would not include all it's css files if it's not in use, but we use Font Awesome in the footer and header, so we need this file to be included manually when Elementor is not used as the editor.

Possible?

We also don't want to include two copies of the same file so some sort of check is needed before including it. 

Here is the file.

wp-content/plugins/elementor/assets/lib/font-awesome/css/font-awesome.min.css?ver=4.7.0

**Comment by ryanlabelle** [comment]
It's actually only some that is not loaded. Try it with:

fa fa-envelope-open-o

**Comment by adiraomj** [comment]
Enqueued `font-awesome` that is part of elementor on all pages and posts. 

This version of font-awesome is the latest version. I did not have any problem with `fa fa-envelope-open-o`. @ryanlabelle please test and let me know if the issue persists. Thanks.

**Comment by ryanlabelle** [comment]
@adiraomj have you tested when Elementor is not used as the main editor? Create a new post and don't use Elementor to edit, save changes using the normal WordPress Editor, see if fa fa-envelope-open-o works in the footer.

**Comment by adiraomj** [comment]
Hey @ryanlabelle, 

Checked the site. We have enqueued elementors font awesome here https://github.com/ryanlabelle/th-widget-pack/blob/master/inc/enqueue.php#L13.

This is a screenshot of the default sample page in embark theme using the envelope icon in the footer: 
![screenshot-themo local-2018 07 12-21-54-08](https://user-images.githubusercontent.com/3372029/42646524-bcb24ca6-861e-11e8-8cd2-1eb7afab5609.png)

Any chance you have a live test site, so that I can test it out. Thanks.
